src/URI/AbsoluteURI.purs

```diff
@@ -124,9 +122,7 @@ _scheme
       (AbsoluteURI userInfo hosts path hierPath query)
       Scheme
 _scheme =
-  lens
-    (\(AbsoluteURI s _ _) → s)
-    (\(AbsoluteURI _ h q) s → AbsoluteURI s h q)
+  lens (_.scheme) (\rec s → rec { scheme = s })
```

Comment:
I think now that we’re using records we can just use the normal (prop (SProxy :: _ “scheme”)) method to make these lenses

Reply:
Dah. I knew there was a better way...

Reply:
Done.